CVE-2024-5129/ubuntu1.yml

```diff
@@ -0,0 +1,59 @@
+- name: Linux target
+  hosts: all
+  become: true
+  tasks:
+    - name: Install unzip
+      ansible.builtin.apt:
+        update_cache: yes
+        name: unzip
+
+    - name: Install requirements
+      ansible.builtin.apt:
+        pkg:
+        - postgresql
+        - postgresql-contrib
+
+    - name: Copy vulnerable source
+      ansible.builtin.unarchive:
+        src: ./data/lunary.zip
+        dest: /tmp
+        mode: '0777'
+
+    - name: Create lunary user for postgres
+      ansible.builtin.shell: >
+        sudo -i -u postgres psql -c "CREATE USER lunary WITH PASSWORD 'lunary'; ALTER ROLE lunary SUPERUSER;"
+    
+    - name: Create lunary db
+      ansible.builtin.shell: >
+        sudo -i -u postgres createdb lunary
+
+    - name: Install nvm
+      ansible.builtin.shell: >
+        curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.39.1/install.sh | bash
+      args:
+        executable: /bin/bash
+        chdir: "$HOME"
+        creates: "$HOME/.nvm/nvm.sh"
+
+    - name: Install node and project deps
+      shell: >
+        . $HOME/.nvm/nvm.sh && nvm install 20.18.0 && cd /tmp/lunary && npm install && npm run migrate:db
```

Comment:
Does "nvm install 20.18.0" pull something from the Internet? Is it possible to keep this version of that thing in the data folder? Imagine 20.18.0 is not available anymore for download. What do we do?

Reply:
I added a version in the data folder

Reply:
What I meant is: please not only put the archive to the data folder, but also install it from the file. My mistake, sorry if it wasn't obvious.